Kbs.Wpf/User/Registration/RegistrationWindow.xaml.cs

```diff
@@ -0,0 +1,101 @@
+using Kbs.Business.Session;
+using Kbs.Business.User;
+using System.Windows;
+using System.Windows.Controls;
+using Kbs.Data.User;
+using Kbs.Wpf.User.Login;
+
+namespace Kbs.Wpf.User.Registration;
+
+public partial class RegistrationWindow : Window
+{
+    
+    private RegistrationViewModel ViewModel => (RegistrationViewModel)DataContext;
+    private readonly UserValidator _userValidator = new UserValidator();
+    private readonly LoginWindow _loginWindow;
+    private readonly IUserRepository _userRepository = new UserRepository();
+    public RegistrationWindow(LoginWindow loginWindow)
+    {
+        _loginWindow = loginWindow;
+        InitializeComponent();
+    }
+    
+    private void SubmitButtonClicked(object sender, RoutedEventArgs e)
+    {
+        var user = new UserEntity()
+        {
+            Email = ViewModel.Email,
+            Name = ViewModel.Name,
```

Comment:
Dit is je probleem: je stelt de Name in op de `ViewModel.Name`, maar `ViewModel.Name` is `null`, dus wordt Name null, en wordt de Db boos. Dit doe je om het op te lossen:
```suggestion
            Name = ViewModel.Name ?? string.Empty,
```